tools/cli/lib/help.c

```diff
@@ -80,20 +82,23 @@ TDNFCliShowHelp(
     printf("upgrade-to                Upgrade a package on your system to the specified version\n");
 }
 
-void
+uint32_t
 TDNFCliShowNoSuchCommand(
-    const char* pszCmd
+    const char *pszCmd
     )
 {
-    printf("No such command: %s. Please use /usr/bin/tdnf --help\n", pszCmd);
+    printf("No such command: %s. Please use /usr/bin/tdnf --help\n",
+            pszCmd ? pszCmd : "");
+    return ERROR_TDNF_CLI_NO_SUCH_CMD;
```

Comment:
If there is a change in the return code, please take care of the harness tests before this change is consumed in photon.